lib/tennis_game.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# TennisGame: used to manage the scoring in a tennis game
+class TennisGame
+  TENNIS_POINTS = %w(Love Fifteen Thirty Fourty).freeze
+  attr_accessor :points
+  def initialize
```

Comment:
Add some spacing between `TENNIS_POINTS` and `attr_accessor` etc - it helps in terms of readability.